spells.yml

```diff
@@ -9949,6 +10029,27 @@ summon_monstrosity:
     - conjuration
     - warlock
     - wizard
+summon_plant:
```

Comment:
Just to clarify, for this spell and its greater variant, is it intended for them to have no class tags? I.e. for them to be subclass-specific spells?

Reply:
That was the intent, since they are summon spells. I figure a plant-based druid subclass will get them eventually

Reply:
Fair enough.